_posts/2020-11-21-a-report-about-presto-conference-tokyo-2020.md

```diff
@@ -0,0 +1,119 @@
+
+# Support Presto as a feature of SaaS
+
+Then, Satoru Kamikaseda, Technical Support Engineer at Treasure Data, provides an overview of how Treasure Data supports Presto in their service. 
+Presto is heavily used to support many enterprise use cases as a customer data platoform, 
+and it is becoming the hub component processing high throughput workload from many kinds of clients such as Spark, ODBC and JDBC.
+
+He describes a stats of Presto inquieries on their platform, and how to support each cases. 
+In the stats, 1/3 is any investigation of job failure and query result, 1/3 is a request to help their client's sql, 
+and others are a sort of notifications to their clients and performance investigation. 
+His talk must be useful for any SaaS companies that provides a query engine to thier clients to learn how hard to support a distibuted query engine.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/GR6e3dfKKJ8w4c" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/SatoruKamikaseda/support-presto-as-a-feature-of-saas" title="Support Presto as a feature of SaaS" target="_blank">Support Presto as a feature of SaaS</a> </strong> from <strong><a href="https://www.slideshare.net/SatoruKamikaseda" target="_blank">SatoruKamikaseda</a></strong> </div>
+
+# How to use Presto with AWS efficiently
+
+We could learn how to use Presto with AWS including Presto on EMR, Presto on EC2, Presto by Athena and AWS Glue.
+Noritaka Sekiyama, Sr. Big Data Architect at Amazon Web Service, Japan, also shares a comparison of Presto on AWS (EC2, EMR, Athena). 
+If you are a new to Presto, his talk gives you an insight to choose your first Presto environement.
+
+<iframe src="//www.slideshare.net/slideshow/embed_code/key/GR6e3dfKKJ8w4c" width="595" height="485" frameborder="0" marginwidth="0" marginheight="0" scrolling="no" style="border:1px solid #CCC; border-width:1px; margin-bottom:5px; max-width: 100%;" allowfullscreen> </iframe> <div style="margin-bottom:5px"> <strong> <a href="//www.slideshare.net/SatoruKamikaseda/support-presto-as-a-feature-of-saas" title="Support Presto as a feature of SaaS" target="_blank">Support Presto as a feature of SaaS</a> </strong> from <strong><a href="https://www.slideshare.net/SatoruKamikaseda" target="_blank">SatoruKamikaseda</a></strong> </div>
+
+# Presto @ LINE 2020
+
+LINE is the biggest company providing the mobile communication tool in Japan (say WhatsApp in Japan). HYuya Ebihara, one of Presto mainteners, 
+gives us how they improve Presto at their platform since they presented in the previous conference (https://prestosql.io/blog/2019/07/11/report-for-presto-conference-tokyo.html). 
```

Comment:
use the internal link reference to link to old post

Reply:
We can use `[the previous conference]({% post_url 2019-07-11-report-for-presto-conference-tokyo %})`

Reply:
thanks.